core/src/test/java/org/elasticsearch/index/mapper/geo/GeoPointFieldMapperTests.java

```diff
@@ -472,30 +476,109 @@ public void testLonLatArrayArrayStored() throws Exception {
+
+        try {
+            String validateMapping = XContentFactory.jsonBuilder().startObject().startObject("type")
+                    .startObject("properties").startObject("point").field("type", "geo_point").field("lat_lon", true).field("geohash", true)
+                    .field("validate_lon", true).endObject().endObject()
+                    .endObject().endObject().string();
+            parser.parse(validateMapping);
+            Assert.fail("process completed successfully when " + MapperParsingException.class.getName() + " expected");
+        } catch (Exception e) {
+            assert(e.getClass().equals(MapperParsingException.class)):
+                    "expected " + MapperParsingException.class.getName() + " but found " + e.getClass().getName();
+        }
+
+        // test deprecated normalize
+        try {
+            String normalizeMapping = XContentFactory.jsonBuilder().startObject().startObject("type")
+                    .startObject("properties").startObject("point").field("type", "geo_point").field("lat_lon", true).field("geohash", true)
+                    .field("normalize", true).endObject().endObject()
+                    .endObject().endObject().string();
+            parser.parse(normalizeMapping);
+            Assert.fail("process completed successfully when " + MapperParsingException.class.getName() + " expected");
+        } catch (Exception e) {
+            assert(e.getClass().equals(MapperParsingException.class)):
+                    "expected " + MapperParsingException.class.getName() + " but found " + e.getClass().getName();
+        }
+
+        try {
+            String normalizeMapping = XContentFactory.jsonBuilder().startObject().startObject("type")
+                    .startObject("properties").startObject("point").field("type", "geo_point").field("lat_lon", true).field("geohash", true)
+                    .field("normalize_lat", true).endObject().endObject()
+                    .endObject().endObject().string();
+            parser.parse(normalizeMapping);
+            Assert.fail("process completed successfully when " + MapperParsingException.class.getName() + " expected");
+        } catch (Exception e) {
+            assert(e.getClass().equals(MapperParsingException.class)):
+                    "expected " + MapperParsingException.class.getName() + " but found " + e.getClass().getName();
+        }
+
+        try {
+            String normalizeMapping = XContentFactory.jsonBuilder().startObject().startObject("type")
+                    .startObject("properties").startObject("point").field("type", "geo_point").field("lat_lon", true).field("geohash", true)
+                    .field("normalize_lon", true).endObject().endObject()
+                    .endObject().endObject().string();
+            parser.parse(normalizeMapping);
+            Assert.fail("process completed successfully when " + MapperParsingException.class.getName() + " expected");
+        } catch (Exception e) {
```

Comment:
I don't understand what this is testing. Don't we want backcompat checks? You can set index version created in settings before creating the index, and you should see the "ok" behavior (old setting is ok, new behavior of coerce or ignoreMalformed), and you can also check that it is serialized back out with the new setting.